k8s/hamm-api-server/network-policy.yaml.ctmpl

```diff
@@ -0,0 +1,14 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: allow-outbound-hamm-api-server
+spec:
+  policyTypes:
+  - Egress
+  podSelector:
+    matchLabels:
+      service: hamm-api-server
+      environment: {{env "ENVIRONMENT"}}
```

Comment:
Could you remove the `project` and `environment` labels here and anywhere else you notice them? I'm trying to get rid of them because I don't think they actually buy us anything